.ci/Jenkinsfile

```
@@ -47,44 +46,44 @@ pipeline {
       steps {
         script {
           dir("${BASE_DIR}/packages") {
-            findFiles()?.collect{ it.name }?.sort()?.each {
-              dir("${it}"){
-                sh(label: "Check integration: ${it}", script: '../../build/elastic-package check -v')
-              }
-            }
-          }
-        }
-        checkGitDiff()
-      }
-    }
-    stage('Test integrations') {
-      steps {
-        dir("${BASE_DIR}") {
-          sh(label: "Boot up the Elastic stack", script: 'build/elastic-package stack up -d -v')
-        }
-        script {
-          dir("${BASE_DIR}/packages") {
+            def integrations = [:]
             // Include hack to skip temporary files with "@tmp" suffix.
             // For reference: https://issues.jenkins.io/browse/JENKINS-52750
             findFiles()?.findAll{ !it.name.endsWith('@tmp') }?.collect{ it.name }?.sort()?.each {
-              dir("${it}"){
-                sh(label: "Test integration: ${it}", script: '''
-                eval "$(../../build/elastic-package stack shellinit)"
-                ../../build/elastic-package test -v --report-format xUnit --report-output file
-                ''')
+              integrations["${it}"] = {
+                env.NODE_LABELS = "master linux x86_64"
```

not sure what this for

```suggestion
                env.NODE_LABELS = "master linux x86_64"
```

---

Firstly it complained about arch, then os and then missing Go runtime. I will try your suggestion with `node('ubuntu-18 && immutable')`